src/migrations/20230208093942-assign-project-api-token-permissions-member.js

```diff
@@ -0,0 +1,20 @@
+exports.up = function (db, cb) {
+    db.runSql(
+        `
+            INSERT INTO role_permission (role_id, permission_id)
+            SELECT (SELECT id as role_id from roles WHERE name = 'Member' LIMIT 1),
+                   p.id   as permission_id
+            FROM permissions p
+            WHERE p.permission='READ_PROJECT_API_TOKEN';
```

Comment:
Should members not have the ability to CREATE/DELETE? Something to discuss at least.